packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,86 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  /* 1 */
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  z-index: 3;
```

Comment:
Do we need this `z-indez`?

Reply:
Yup you're totally right, this doesn't serve a purpose! 😄